staff_features/accessions/step_definitions/accession_edit.rb

```diff
@@ -0,0 +1,188 @@
+# frozen_string_literal: true
+
+Given 'the Accession appears in the search results list' do
+  visit "#{STAFF_URL}/accessions"
+
+  fill_in 'filter-text', with: @uuid
+
+  within '.search-filter' do
+    find('button').click
+  end
+
+  search_result_rows = all('#tabledSearchResults tbody tr')
+  expect(search_result_rows.length).to eq 1
+
+  within search_result_rows[0] do
+    element = find('a', text: 'Edit')
+
+    @accession_id = URI.decode_www_form_component(element[:href]).split('/').pop
+  end
+end
+
+Given 'the Accession is opened in the view mode' do
+  visit "#{STAFF_URL}/accessions"
+
+  fill_in 'filter-text', with: @uuid
+
+  within '.search-filter' do
+    find('button').click
+  end
+
+  search_result_rows = all('#tabledSearchResults tbody tr')
+  expect(search_result_rows.length).to eq 1
+
+  within search_result_rows[0] do
+    element = find('a', text: 'View')
+
+    @accession_id = URI.decode_www_form_component(element[:href]).split('/').pop
+  end
+
+  click_on 'View'
+end
+
+Given 'the Accession is opened in edit mode' do
+  visit "#{STAFF_URL}/accessions"
+
+  fill_in 'filter-text', with: @uuid
+
+  within '.search-filter' do
+    find('button').click
+  end
+
+  search_result_rows = all('#tabledSearchResults tbody tr')
+  expect(search_result_rows.length).to eq 1
+
+  within search_result_rows[0] do
+    element = find('a', text: 'Edit')
+
+    @accession_id = URI.decode_www_form_component(element[:href]).split('/').pop
+  end
+
+  click_on 'Edit'
+end
+
+When 'the user changes the {string} field to {string}' do |field, value|
+  fill_in field, with: value
+end
+
+When 'the user changes the {string} field' do |field|
+  fill_in field, with: "Updated Title #{SecureRandom.uuid}"
+end
+
+Then 'the {string} field has the original value' do |field|
+  expect(page).to have_field(field, with: "Accession Title #{@uuid}")
```

Comment:
Same comment as previous - function name should match the behavior, which is specific to the Title field.